map/generate-offline-map-overrides/src/main/java/com/esri/samples/generate_offline_map_overrides/GenerateOfflineMapOverridesController.java

```diff
@@ -147,7 +147,7 @@ private void generateOfflineMap() {
               // get the export tile cache parameters for the base layer
               OfflineMapParametersKey basemapParamKey = new OfflineMapParametersKey(
                   mapView.getMap().getBasemap().getBaseLayers().get(0));
-              ExportTileCacheParameters exportTileCacheParameters =
+              var exportTileCacheParameters =
```

Comment:
Would argue this one as not being a `new ExportTileCacheParameters` but up to you!